### DISCUSSION TITLE
Data export / Preview: per customer / per activity?

### DISCUSSION THREAD

**Discussion by uku69** [kimai/kimai#4989] [General]
Hi folks,
if I prepare a general data export in kimai, i get a preview of hours per customer. That's fine, but I miss a possibilty to see the hours per activity. In the report dashboard I get the information, but only within a month - and not for a free selection. In the "yearly report" I see the data also in this granularity, but there i can't select over the export status.
So do you have any hints for that request without exporting the complete data to excel and make the examination there?
Thanks a lot in advance! 
uku69.

**Comment by empx92** [comment]
We simply wrote our own export templates, and as far as i know you should see every activity if there is a entry in the timesheet?
